lib/Transform/operators_supported.json

```diff
@@ -511,6 +511,10 @@
 	    "properties" : [
 		["Storage.Bytes", "Total storage required for matmul engine", "long"]
 	    ]
-	}	
+	},
+	{
+	    "name" : "xten.globalaveragepool2d",
```

Comment:
If you want to get the visualizer working in the same PR, you should also need a change in `lib/Transform/ATenVisualGraph.cpp`. See, for example, #17.

Reply:
`xten.globalaveragepool2d` does not hold any properties. Added it nevertheless in case we update it in the future.